packages/components/Notification/src/NotificationTokens.ts

```diff
@@ -1,6 +1,7 @@
 import { Theme } from '@fluentui-react-native/framework';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { NotificationTokens } from './Notification.types';
+import { notificationColors } from './NotificationTokens.ios';
 
 export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme> = (t: Theme) =>
```

Comment:
Since this file won't be used by iOS anymore, you can remove the iOS only props from line 14 and 20.